docs/hooks.md

```diff
@@ -0,0 +1,33 @@
+## Plugin Hooks
+SparkPost's WordPress plugin has a number of hooks that can be used to modify how it works and/or inject new functionality around its lifecycle.
+
+### Convention
+Hook names are prefixed with `wpsp_`.There are some hooks that are common across mailers (http and smtp). Unless the hook applies to both mailer, it has either `smtp_` or `smtp_` after the initial prefix (`wpsp_`).
+
+### List of hooks
+
+| Hook Name                         | Applicable Mailer          | Description (Purpose)  |
+| -------------                     |:----------------:| -----:|
+| wpsp_get_settings                 | Both | Tap into settings objects  
+| wpsp_init_http_mailer*            | HTTP | Modify/replace http mailer instance
```

Comment:
This is the only hook that doesn't follow the convention listed where the `smtp` or `http` directly follows the `wpsp`